components/Favorites/Favorites.module.css

```diff
@@ -0,0 +1,21 @@
+
+.favbuttonoff {
```

Comment:
camelCase? not important though

```suggestion
.favButtonOff {
```